bin/docker-entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+GC_OPTS="-XX:+UseConcMarkSweepGC -XX:MaxPermSize=512m -XX:+CMSClassUnloadingEnabled"
```

Comment:
These are defaults taken from the `spark-jobserver` [launch script](https://github.com/spark-jobserver/spark-jobserver/blob/master/bin/server_start.sh). I trimmed it down because most of the guard are things we are in control of by building the image.